packages/cli/commands/project/watch.js

```diff
@@ -0,0 +1,69 @@
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const {
+  getProjectConfig,
+  validateProjectConfig,
+  pollBuildStatus,
+  pollDeployStatus,
+} = require('../../lib/projects');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { createWatcher } = require('@hubspot/cli-lib/projectsWatch');
+
+const i18nKey = 'cli.commands.project.subcommands.watch';
+
+exports.command = 'watch [path]';
+exports.describe = false;
+
+const handleBuildStatus = async (accountId, projectName, buildId) => {
+  const {
+    isAutoDeployEnabled,
+    deployStatusTaskLocator,
+  } = await pollBuildStatus(accountId, projectName, buildId);
+
+  if (isAutoDeployEnabled && deployStatusTaskLocator) {
+    await pollDeployStatus(
+      accountId,
+      projectName,
+      deployStatusTaskLocator.id,
+      buildId
+    );
+  }
+};
```

Comment:
The polling status function uses a lot of CLI related stuff that doesn't exist in CLI-LIB (spinnies, for instance) so this is really just an effort to avoid adding new deps to CLI-LIB